monitor/main.go

```diff
@@ -176,7 +179,7 @@ func blockProcessor(ctx context.Context, startBlock *big.Int, headers <-chan *ty
 
 		parseAndLogEvent := func(logger log.Logger, eventIdx int, eventLog *types.Log) {
 			eventLogger := logger.New("logTxIndex", eventIdx, "logBlockIndex", eventLog.Index)
-			parsed, err := r.TryParseLog(transactionCtx, eventLog, h.Number)
+			parsed, err := r.TryParseLog(transactionCtx, *eventLog, h.Number)
```

Comment:
why this change?
doesn't look like you changed the signature in the [kliento PR](https://github.com/celo-org/kliento/pull/10)

Reply:
Looks like there was a change to Kliento that Eksportisto had not yet adopted that changed the eventLog from a pointer to a value:
https://github.com/celo-org/kliento/commit/feecb3948ea292a2d81d2e668c092f195efca810#diff-5593a38c8a92809ab9448ee33f120e244e7fc4c0fd0689e239a9e85dc4ae0933L44